pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -1603,3 +1604,25 @@ def test_agg_with_as_index_false_with_list():
         columns=MultiIndex.from_tuples([("a1", ""), ("a2", ""), ("b", "sum")]),
     )
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.skipif(
+    not typing.TYPE_CHECKING, reason="TYPE_CHECKING must be True to import pyarrow"
```

Comment:
the reason here doesn't seem right to me.  are you sure?

Reply:
I have updated it to make the reason more clear. Please review it.